src/_assets/page_assets/tools/android_studio.svg

```diff
@@ -1,811 +1,46 @@
-						c-0.1,1,0.5,2.1,1.8,3.4L343,74.5c1.6,1.6,3.7,2.4,5.7,2.4v-0.1l0.1,0.1c2.1,0,4.2-0.8,5.7-2.4l23.1-23.1
-						c1.3-1.3,1.8-2.5,1.8-3.4h0.1V22C379.4,15.6,374.3,10.4,367.9,10.4z M348.8,51.3h-0.1l0,0c-8.4,0-15.3-6.9-15.3-15.4
-						c0-8.5,6.9-15.4,15.3-15.4v-0.2l0.1,0.2c8.4,0.1,15.3,6.9,15.3,15.4S357.2,51.2,348.8,51.3z"/>
-					<path opacity="0.2" fill="#FFFFFF" enable-background="new    " d="M361.5-8.9v-1.2v-0.1h-12.8h-12.7v0.1v1.2h12.7H361.5z"/>
-					<path fill="#FFFFFF" fill-opacity="0.2" d="M348.8,50h-0.1l0,0c-8.2,0-14.9-6.6-15.3-14.7c0,0.2-0.1,0.4-0.1,0.6
-						c0,8.4,6.9,15.4,15.3,15.4l0,0h0.1c8.4-0.1,15.3-6.9,15.3-15.4c0-0.2,0-0.4-0.1-0.6C363.7,43.4,357,49.9,348.8,50z"/>
-					<path opacity="0.2" fill="#FFFFFF" enable-background="new    " d="M329.6,11.7h6.5v-1.3h-6.5c-6.3,0-11.5,5.2-11.5,11.5v1.3
-						C318,16.9,323.2,11.7,329.6,11.7z"/>
-					<path opacity="0.2" fill="#FFFFFF" enable-background="new    " d="M367.9,10.4h-6.4v1.3h6.4c6.3,0,11.5,5.2,11.5,11.5V22
-						C379.4,15.6,374.3,10.4,367.9,10.4z"/>
-					<path opacity="0.2" fill="#262626" enable-background="new    " d="M348.8,20.6l-0.1-0.2v0.2c-8.4,0-15.3,6.9-15.3,15.4
-						c0,0.2,0,0.4,0.1,0.6c0.4-8.2,7.1-14.7,15.3-14.7v-0.2l0.1,0.2c8.2,0.1,14.9,6.6,15.2,14.7c0-0.2,0.1-0.4,0.1-0.6
-						C364.1,27.5,357.2,20.6,348.8,20.6z"/>
-					<path fill="#262626" fill-opacity="0.2" d="M377.6,50.2l-23.1,23.1c-1.6,1.6-3.7,2.4-5.7,2.4l-0.1-0.1v0.1
-						c-2.1,0-4.2-0.8-5.7-2.4l-23.1-23.1c-1.3-1.3-1.8-2.5-1.8-3.4H318v1.3h0.1c-0.1,1,0.5,2.1,1.8,3.4L343,74.5
-						c1.6,1.6,3.7,2.4,5.7,2.4v-0.1l0.1,0.1c2.1,0,4.2-0.8,5.7-2.4l23.1-23.1c1.3-1.3,1.8-2.5,1.8-3.4h0.1v-1.3h-0.1
-						C379.4,47.7,378.9,48.9,377.6,50.2z"/>
-					<path fill="none" d="M348.7,51.3L348.7,51.3h0.1c8.4-0.1,15.3-6.9,15.3-15.4s-6.9-15.3-15.3-15.4l-0.1-0.2v0.2
-						c-8.4,0-15.3,6.9-15.3,15.4C333.4,44.4,340.2,51.3,348.7,51.3z"/>
-				</g>
-			</g>
-		</g>
-		
-			<radialGradient id="SVGID_19_" cx="-48.0896" cy="205.591" r="1061.7498" gradientTransform="matrix(0.2559 0 0 -0.2559 283.5 33.5176)" gradientUnits="userSpaceOnUse">
-			<stop  offset="0" style="stop-color:#FFFFFF"/>
-			<stop  offset="1" style="stop-color:#FFFFFF;stop-opacity:0"/>
-		</radialGradient>
-		<path fill="url(#SVGID_19_)" fill-opacity="0.1" d="M436,163.6L436,163.6l-16.4-28.3c23.9-20.2,39.1-50.3,39.1-84
-			c0-60.7-49.3-110-110-110s-110,49.3-110,110c0,33.7,15.2,63.9,39.1,84.1l-16.4,28.3c-0.5,0.9-0.8,1.8-0.9,2.8l-1.4,18.5
-			c-0.1,0.7,0.3,1.3,0.9,1.7l0,0l0,0c0.6,0.4,1.3,0.3,1.9-0.1l15.4-10.4c0.8-0.6,1.5-1.3,2-2.1l0,0l15.5-26.8
-			c15.9,9,34.3,14.1,53.9,14.1s38-5.2,54-14.2l15.5,26.8c0.5,0.9,1.2,1.6,2,2.1l15.4,10.5c0.6,0.4,1.3,0.4,1.9,0.1l0,0l0,0
-			c0.6-0.4,0.9-1,0.9-1.7l-1.4-18.5C436.8,165.5,436.6,164.5,436,163.6z"/>
-	</g>
-	<rect x="225.9" y="-58.7" fill="none" width="245.6" height="245.6"/>
-</g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="48px" height="48px" viewBox="0 0 192 192">
```

Comment:
This is a simple logo and yet, this SVG is 350K in size (which is huge for an SVG) because it contains embedded image elements. I don't think that it should. Maybe double check with Richard?

/cc @kwalrath

Reply:
The icon we need here, so I just swapped it out by the png from go/icons. That's 3 kb.